int16 counter
string lcdA
string lcdB
string lcdC
string lcdD
int16 foo
